src/parser/jobs/blm/modules/Triple.js

```diff
@@ -53,8 +54,25 @@ export default class Triple extends Module {
 			return
 		}
 
-		// Save the event to the DWT casts
-		this._triple.casts.push(event)
+		//stop tracking
```

Comment:
drop me a line on discord, not sure I follow why triple is being closed in the cast hook